images/macos/macos-11-Readme.md

```diff
@@ -79,7 +80,7 @@
 - mongo v5.0.2
 - mongod v5.0.2
 - Newman 5.3.0
-- OpenSSL 1.1.1l  24 Aug 2021 `(/usr/local/opt/openssl -> /usr/local/Cellar/openssl@1.1/1.1.1l)`
+- OpenSSL 1.1.1l  24 Aug 2021 `(/usr/local/opt/openssl -> /usr/local/Cellar/openssl@3/3.0.0)`
```

Comment:
Please check default openssl version

Reply:
@al-cheb @miketimofeev here is the pull request that fixes this behaviour https://github.com/actions/virtual-environments/pull/4153